javascript/javascript1/week2/fullname.js

```diff
@@ -0,0 +1,32 @@
+//formal fullname
+
+function getFullname(firstname, surname, userformalName) {
+  if (userformalName === true) {
+    return "lord" + firstname + surname;
+  } else {
+    return firstname + surname;
+  }
+}
+
+const fullname3 = getFullname("Benjamin", "Hughes", true);
+console.log(fullname3);
```

Comment:
this will print all characters continuously without space. the output will be  'lordBenjaminHughes". Ideally should be 'lord Benjamin Hughes'.